### PR TITLE
New post: Server Software

### PR DESCRIPTION
**Pull request by Lovinoes**
I wrote a guide on some popular Minecraft server jars. Minecraft is our beloved game, so feel free to suggest more jars or improvements! The guide provides a complete detailed comparison of options like Pufferfish, Paper, Purpur and so on... 🙂

### CODE REVIEW

File: content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+
+<!-- Hybrid Servers: Mohist, Magma, Arclight -->
+
+
+## Hybrid Servers
+Hybrid servers combine the capabilities of both modded and plugin-based servers, allowing for a versatile environment where both can coexist.
+
+### Mohist
+Mohist is a server jar that combines the capabilities of Forge and Bukkit, allowing for the use of both Forge mods and Bukkit plugins. It’s ideal for servers that need to blend the extensive modding capabilities of Forge with the versatility of Bukkit plugins.
```
Comment (iamdeka) [inline content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md:90]: https://essentialsx.net/do-not-use-mohist.html
https://github.com/EssentialsX/Website/pull/44
https://github.com/EssentialsX/Essentials/pull/3580

File: content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+<!-- END -->
+
+## Conclusion
+- **For Optimal Performance**: Choose **Pufferfish** for a performance-focused server that remains close to Paper's behavior.
+- **For a Stable and Supported Server**: Opt for **Paper** if you want a balance of performance and compatibility with a wide range of plugins.
+- **For Extra Features**: Use **Purpur** if you want additional gameplay features and customization while maintaining the performance base of Pufferfish.
+- **For Basic Performance and Compatibility**: Use **Spigot** if you need a reliable server with good plugin support.
+- **For a Plugin-Only Server**: Choose **SpongeVanilla** if you want to use Sponge plugins without the need for Forge mods, ensuring a streamlined plugin experience.
+- **For Modern Modding**: Choose **Fabric** for a large modding ecosystem with performance benefits, **Forge** for extensive mod support and classic modded gameplay, or **SpongeForge** for integrating Sponge plugins with Forge mods.
+- **For Combining Plugins and Mods**: Consider **Mohist**, **Magma**, **Arclight**, **NeoForge**, or **Quilt** based on your specific needs for modding and plugin compatibility.
```
Comment (iamdeka) [inline content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md:144]: 👎

File: content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+### Velocity
+Velocity is the modern, high-performance proxy designed with performance and stability in mind. It’s a full alternative to Waterfall with its own plugin ecosystem.
+- **High Performance**: Velocity is known for its low-latency performance, making it ideal for large, busy networks.
+- **Enhanced Security**: It includes advanced security features, helping to protect your network from attacks and exploits.
+- **Active Development and Ecosystem**: Velocity is actively maintained and has its own plugin ecosystem, making it a forward-looking choice for modern Minecraft server networks.
+
+<!-- END -->
+
+## Conclusion
```
Comment (iamdeka) [inline content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md:137]: I would suggest against promoting the usage of hybrid server software (except SpongeForge, i guess)

File: content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+- **For Optimal Performance**: Choose **Pufferfish** for a performance-focused server that remains close to Paper's behavior.
+- **For a Stable and Supported Server**: Opt for **Paper** if you want a balance of performance and compatibility with a wide range of plugins.
+- **For Extra Features**: Use **Purpur** if you want additional gameplay features and customization while maintaining the performance base of Pufferfish.
+- **For Basic Performance and Compatibility**: Use **Spigot** if you need a reliable server with good plugin support.
+- **For a Plugin-Only Server**: Choose **SpongeVanilla** if you want to use Sponge plugins without the need for Forge mods, ensuring a streamlined plugin experience.
```
Comment (iamdeka) [inline content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md:142]: > For a Plugin-Only Server

⁉️

File: content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+- **For Modern Modding**: Choose **Fabric** for a large modding ecosystem with performance benefits, **Forge** for extensive mod support and classic modded gameplay, or **SpongeForge** for integrating Sponge plugins with Forge mods.
+- **For Combining Plugins and Mods**: Consider **Mohist**, **Magma**, **Arclight**, **NeoForge**, or **Quilt** based on your specific needs for modding and plugin compatibility.
+- **For Networking Multiple Servers**: Use **BungeeCord** or **Waterfall** for traditional proxy setups, or **Velocity** if you need advanced performance and security features.
```
Comment (iamdeka) [inline content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md:145]: BungeeCord is considered legacy and most server administrators should use Velocity or at least Waterfall instead

File: content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+- **Enhanced Security**: It includes advanced security features, helping to protect your network from attacks and exploits.
+- **Active Development and Ecosystem**: Velocity is actively maintained and has its own plugin ecosystem, making it a forward-looking choice for modern Minecraft server networks.
+
+<!-- END -->
+
+## Conclusion
+- **For Optimal Performance**: Choose **Pufferfish** for a performance-focused server that remains close to Paper's behavior.
+- **For a Stable and Supported Server**: Opt for **Paper** if you want a balance of performance and compatibility with a wide range of plugins.
+- **For Extra Features**: Use **Purpur** if you want additional gameplay features and customization while maintaining the performance base of Pufferfish.
+- **For Basic Performance and Compatibility**: Use **Spigot** if you need a reliable server with good plugin support.
```
Comment (iamdeka) [inline content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md:141]: Spigot doesn't support Paper plugins (duh)

**Review by iamdeka — CHANGES_REQUESTED** [review]

File: content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md (hunk 6 above)
Comment (Lovinoes) [reply content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md:141]: > Spigot doesn't support Paper plugins (duh)

It doesn't say anywhere that it supports Paper plugins. 99% of plugins out there are built for either Bukkit or Spigot. (duh)

File: content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md (hunk 5 above)
Comment (Lovinoes) [reply content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md:145]: > BungeeCord is considered legacy and most server administrators should use Velocity or at least Waterfall instead.

Good point. I'll edit that

File: content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md (hunk 6 above)
Comment (iamdeka) [reply content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md:141]: "For Basic Performance and **Compatibility**"

Comment (Lovinoes) [reply content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md]: my bad

**Comment by Lovinoes** [comment]
Adjusted the things with Mohist, Spigot, "Plugin-Only Server", and hybrid servers. I hope it's good now 🤷‍♂️

**Comment by Lovinoes** [comment]
I'm in the process of reorganizing some things. Please don't merge *if accepted* ^_____^

**Comment by Lovinoes** [comment]
It's ready imo. You might want to add a few more "warnings" because Idfk how to mark some notes properly.

**Comment by iamdeka** [comment]
deleted the file i made changes to and made a new one 😄

**Comment by Lovinoes** [comment]
> deleted the file i made changes to and made a new one 😄

I am gigachad I know

File: content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md (hunk 5 above)
Comment (Lovinoes) [reply content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md]: Done

**Comment by iamdeka** [comment]
Thought you were gonna do it for Bedrock software too, godspeed

**Review by Mocab — APPROVED** [review]

**Review by iamdeka — APPROVED** [review]

**Pull request merged**